WordPress/twentytwentyone
# Adds word break to latest posts grid view

Fixes #678 

## Summary

Added `word-wrap: anywhere` to the latest post grid view to ensure long words in the title and excerpt are breaking if the space is too narrow. 

## Relevant technical choices:

@melchoyce suggested `word-wrap: break-word`, but this only worked for the excerpts but not the title.

## Test instructions

This PR can be tested by following these steps:
1. Create a new post or update one of the latest posts with a long title and a long word inside the excerpt.
2. Create a new post or page and add the latest posts block, choose the grid view, do not use wide or full width on this block.
3. Check the post or page on the frontend.


## Screenshots

If this is a visual change please include screenshots of the change at various screen sizes.

**Before**
![grafik](https://user-images.githubusercontent.com/7585600/97083223-e1e27500-160e-11eb-976a-c8e22093d381.png)

**After**
![grafik](https://user-images.githubusercontent.com/7585600/97083203-d4c58600-160e-11eb-8f75-c3fd19ed714e.png)


## Quality assurance
